src/olympia/stats/utils.py

```diff
@@ -192,30 +194,38 @@ def get_averages_by_addon_from_bigquery(today, exclude=None):
     `{get_amo_stats_dau_view_name()}`
   WHERE
     submission_date >= @one_week_date
+  AND
+    EXTRACT(DAYOFWEEK FROM submission_date) <> 1
+  AND
+    EXTRACT(DAYOFWEEK FROM submission_date) <> 7
```

Comment:
Lets hope there aren't any weekend focused extensions! 😬

Reply:
Since this is just data about which users have the add-on activated and using Firefox, I guess the real problem is week-end focused _users_. If some users are only using Firefox during the week-end, and never pinging during the rest of the week at all, they would be ignored. We're trying to avoid the opposite problem (low week-ends activity skewing the data) so hopefully that's ok.